packages/x-date-pickers/src/ClockPicker/ClockPicker.tsx

```diff
@@ -159,7 +159,7 @@ export const ClockPicker = React.forwardRef(function ClockPicker<TDate extends u
     components,
     componentsProps,
     value,
-    disableIgnoringDatePartForTimeValidation,
+    disableIgnoringDatePartForTimeValidation = false,
```

Comment:
```suggestion
    disableIgnoringDatePartForTimeValidation,
```

Reply:
Same as above but for https://github.com/mui/mui-x/pull/6667/files#diff-e2061e7eec043ba1ab45746803975267576725a37d32733b469fc7f8b2079a00L40.

Happy to revert and ignore, but this felt quite safe as it didn't seem to be publicly exposed.

Reply:
If `eslint` rules call for it, IMHO, it was a perfectly fine change.
We already default it as `false` in the function down the line.

Reply:
I'll make a separate PR. That way we have an easy to bissect git history should it run into an unforeseen edge case